src/github_sdk.py

```diff
@@ -1,101 +1,134 @@
 import gzip
 import io
 import logging
-import os
 import uuid
 from datetime import datetime
 
 import requests
 from sentry_sdk.envelope import Envelope
 from sentry_sdk.utils import format_timestamp
 
-logging.basicConfig(level=os.environ.get("LOGGING_LEVEL", "INFO"))
-
-# We need an authorized token to fetch the API. If you have SSO on your org you will need to grant permission
-# Your app and the Github webhook will share this secret
-# You can create an .env file and place the token in it
-GH_TOKEN = os.environ.get("GH_TOKEN")
-# Where to report Github actions transactions
-SENTRY_GITHUB_DSN = os.environ.get("SENTRY_GITHUB_DSN")
```

Comment:
Moving this under `handle_event` and we will configure it inside the client's constructor.